alertproduction.tex

```diff
@@ -25,10 +22,11 @@ \section{Alert Production in Commissioning and Early Operations}
 By the end of the commissioning period, coadd templates for use in difference imaging will only be available for $\approx$ 10\% of the sky.
 Generating templates over a wide area is not an explicit goal of commissioning;  however, where possible, if commissioning observations are agnostic to pointing and filter, we would endeavour to choose a pointing and filter that maximizes building templates to enable early science.
 
-Rubin aims to scale up alert production during commissioning with the aim of beginning routine Alert Production as soon as is feasible following System First Light  (\S~\ref{sec:timeline}).
+Rubin aims to scale up alert production during commissioning with the aim of beginning routine Alert Production as soon as is feasible following Rubin First Light  (\S~\ref{sec:timeline}).
 \citeds{RTN-061} describes the criteria for sending the first Rubin alerts.
 Once begun, Alert Production will then proceed continuously into the full LSST survey.
-Alerts generated during commissioning may be produced with higher latency, and access to images and the PPDB may not be immediately available.
+Alerts generated during the early science period may be produced with higher latency, and access to images and the PPDB may not be immediately available.
+During the first phases of Alert Production, alert packets for moving objects might not include the associated historical source records , and parameters such as phase curve slope (G) would be empty until sufficient detections exist to derive them.
 
 Table~\ref{tab:prompt-data-products} lists the various alert and prompt processing data products currently planned at each phase of alert production during commissioning and the first two years of the LSST survey. 
```

Comment:
This description is inconsistent with both the actual table content and with the table caption.

Here it says "each phase" from "commissioning" through "the first two years of the ... survey".

In the table caption it says "during commissioning and year 1 of survey observations".

In the table headers it refers only to "Phase 1: 3 -- 16 weeks post [RFLight, July 4th]" and "Phase 2: 18 -- 17 weeks post RFL". Even apart from the apparent "18---17" typo, this clearly isn't covering routine ops in a clear way.